src/ol_openedx_chat_xblock/ol_openedx_chat_xblock/block.py

```diff
@@ -177,6 +193,15 @@ def ol_chat(self, request, suffix=""):  # noqa: ARG002, PLR0911
             )
             xblock_response = Response(response.content)
 
+            # Sending tracker event for response
+            tracker.emit(
+                "chat.xblock.response",
+                {
+                    **tracker_base_data,
+                    "response_data": str(xblock_response),
```

Comment:
I am not sure if it is useful to log the response like this. In Chat Aside, we log the response message only.